roles/upgrade/README.md

```diff
@@ -29,7 +29,7 @@ Additionally a list of any non-Red Hat RPM packages that were installed on the s
 | post_upgrade_unset_release| true                | Boolean used to control whether Leapp's RHSM release lock is unset. |
 | post_upgrade_release    |                       | Optional string used to set a specific RHSM release lock after the Leapp upgrade, but before the final update pass. |
 | kernel_modules_to_unload_before_upgrade | []    | A list of kernel modules to be unloaded prior to running leapp. |
-| post_7_to_8_python_interpreter | /usr/libexec/platform-python | For RHEL 7 to 8 upgrades, /usr/bin/python is discovered but not available post upgrade. For 7 to 8 upgrades, ansible_python_interpreter is set to this value post upgrade reboot prior to reconnecting. |
```

Comment:
You should add a note that `post_7_to_8_python_interpreter` is Deprecated, use `post_upgrade_python_interpreter` instead.